### PR TITLE
[RDAPI-27721] MimeType Body Factory field

### PR DESCRIPTION
**Pull request by Bcolemanaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

New field "Body Factory" added to Policy Studio Mime Type configuration settings.

## Deploy preview link

https://deploy-preview-2424--axway-open-docs.netlify.app/docs/apim_reference/general_settings/#mime-settings-configuration
https://deploy-preview-2424--axway-open-docs.netlify.app/docs/apim_policydev/apigw_polref/content_additional/#mime-type-settings
https://deploy-preview-2424--axway-open-docs.netlify.app/docs/apim_policydev/apigw_polref/content_common/#configure-mime-types

## Checklist for contributors

Before submitting this PR, please make sure:

_This is simply a reminder of what we are going to look for before merging your change._

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [X] You have verified that all status checks have passed

### CODE REVIEW

File: content/en/docs/apim_reference/general_settings.md [hunk 1]
```diff
@@ -85,9 +85,9 @@ The file extensions are used to set the `Content-Type` headers returned by a [St
...
 
-The MIME settings screen lists the actual MIME types on the left column of the table, together with their corresponding file extensions (where applicable) in the right column.
+The MIME settings screen lists the actual *MIME types* on the left column of the table, the corresponding file *Extensions* (where applicable) in the middle column, and the optional *Body Factories* in the right column.
 
-To add a new MIME type, click the **Add** button. In the **Configure MIME Type** dialog, enter the new content type in the **MIME Type** field. If the new type has a corresponding file extension, enter this extension in the **Extension** field. Click the **OK** button when finished.
+To add a new MIME type, click the **Add** button. In the **Configure MIME Type** dialog, enter the new content type in the **MIME Type** field. If the new type has a corresponding file extension, enter this extension in the **Extension** field. If a body factory is required to better describe how the Mime Type should be handled, it can be selected from the **Body Factory** dropdown list. Click the **OK** button when finished.
```
Comment (scormicanAxway) [inline content/en/docs/apim_reference/general_settings.md:90]: @Bcolemanaxway You use Mime here, elsewhere it's MIME

File: content/en/docs/apim_policydev/apigw_polref/content_common.md [hunk 2]
```diff
@@ -663,7 +663,7 @@ dialog.
 
 Enter a name for the new type in the **MIME Type**
 field, and the corresponding file extension in the **Extension**
-field.
+field. You can optionally select from a list of body factories in the **Body Factory** dropdown to describe how the Mime Type should be handled.
```
Comment (scormicanAxway) [inline content/en/docs/apim_policydev/apigw_polref/content_common.md:666]: Mime vs MIME

File: content/en/docs/apim_policydev/apigw_polref/content_additional.md [hunk 3]
```diff
@@ -132,7 +132,7 @@ option.
 
 You can add new types by selecting the **Add**
 button and entering a type name and corresponding extension on the **Configure MIME Type**
-dialog. You can enter a list of extensions by separating them with spaces. You can edit or delete existing types by selecting the **Edit**
+dialog. You can enter a list of extensions by separating them with spaces. You can optionally select from a list of Body Factories to describe how the Mime Type should be handled. You can edit or delete existing types by selecting the **Edit**
```
Comment (scormicanAxway) [inline content/en/docs/apim_policydev/apigw_polref/content_additional.md:135]: Mime vs MIME

**Review by JohnD-Axway — APPROVED** [review]

**Comment by andreamussap** [comment]
@JohnD-Axway is this a new feature or important change that you can add a placeholder to the release notes already?

**Comment by JohnD-Axway** [comment]
@andreamussap I'm not sure this would really warrant a release note.

**Pull request merged**